components/builder-web/app/actions/oauth.ts

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
This should be 2016-2018

Reply:
Last I heard, it was sufficient to just put the year it was written, so 2018 should be fine.  That's what I've been putting in new files.

Reply:
Thanks, I'll fix this up.